rwengine/src/engine/SaveGame.cpp

```diff
@@ -107,41 +107,41 @@ struct StructWeaponSlot {
     BlockDword unknown2;
 };
 struct StructPed {
-    uint8_t unknown0_[52];
+    uint8_t unknown0_[52]{};
     glm::vec3 position{};
-    uint8_t unknown1[640];
-    float health;
-    float armour;
-    uint8_t unknown2[148];
-    std::array<StructWeaponSlot, kNrOfWeapons> weapons;
-    uint8_t unknown3[348];
+    uint8_t unknown1[640]{};
```

Comment:
nit: it probably better to replace c style array with `std::array`.

Reply:
Are the braces needed here? Do we want a constructor there?

Reply:
Reverted to the original situation.